.eslintrc

```diff
@@ -10,8 +10,8 @@
       "babel-module": {}
     }
   },
-  "plugins": ["mocha", "jest", "va-enzyme", "react-hooks", "fp"],
-  "extends": ["airbnb", "plugin:prettier/recommended"],
+  "plugins": ["mocha", "jest", "va-enzyme", "react-hooks", "fp", "lodash"],
+  "extends": ["airbnb", "plugin:prettier/recommended", "plugin:lodash/recommended"],
```

Comment:
should help block full imports from lodash

Reply:
Sweet! https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/import-scope.md

Reply:
It's not really necessary to restrict full imports, the babel lodash plugin turns full imports into individual ones (as long as you don't use `chain`).

I believe the size of lodash in our bundle is due to `lodash/fp` and because some formation-react components use lodash but we accidentally removed the lodash babel plugin from the config in the vfs tools repo.

And by "we," I mean me. :-)